test/cat.js

```diff
@@ -64,3 +64,56 @@ test('glob', t => {
   t.truthy(result.search('test1') > -1); // file order might be random
   t.truthy(result.search('test2') > -1);
 });
+
+test('without EOF', t => {
+  const result = shell.cat('test/resources/cat/file3');
+  t.falsy(shell.error());
+  t.is(result.code, 0);
+  t.is(result.toString(), 'test3');
+});
+
+test('empty', t => {
```

Comment:
Can you edit the PR description to mention that you're adding 2 additional tests for previous functionality? I assume this is code that worked before--please correct me if that's not so.

Reply:
Yes, it has work fine. I add these tests as part of the `-n` implementation.